dev.sparcles.SREngine.yml

```diff
@@ -0,0 +1,40 @@
+id: dev.sparcles.SREngine
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: srengine-run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+
+modules:
+   - name: srengine
+     buildsystem: cmake-ninja
+     builddir: true
+     config-opts:
+       - -DCMAKE_BUILD_TYPE=Release
+       - -DSR_ENGINE_FLATPAK_BUILD=ON
+       - -DSR_ENGINE_EXTENDED_NAME=OFF
+
+     post-install:
+       - install -Dm755 ../srengine-run.sh /app/bin/srengine-run.sh
+       - mkdir -p /app/share/SREngine
+       - install -Dm644 ../dev.sparcles.SREngine.metainfo.xml -t /app/share/metainfo
+       - install -Dm644 ../Resources/Engine/Textures/dev.sparcles.SREngine -t /app/share/icons/hicolor/400x400/apps/
```

Comment:
400x400 isn't a standard size, it's in powers of 2. Install a 128x128, 256x256 or 512x512

The filename must end in `.png` or `.svg`

Reply:
Also I don't see this file here https://github.com/SpaRcle-Studio/SREngine/tree/dev/Resources/Engine/Textures

Reply:
Done.